### DISCUSSION TITLE
"Io domain"

### DISCUSSION THREAD

**Discussion by RiskySolutions** [jsr-io/jsr#715] [Q&A]
**Hello Everyone**

{ Please excuse me if this question maybe strike you easily to answer & if it maybe be irrelvant }

> Can Someone here explain the 5 Ws behind what the Io.domain really does?

**Comment by halvardssm** [comment]
On that note, would it be an idea to get another TLD as well to have as fallback already built in to jsr? something more "open" like org or net would be good contestants

**Reply by RiskySolutions** [reply]
@halvardssm What does the io domain really do though? there is something more to it then us developers see happening because alot of companies with many people are switching to this domain?

**Reply by halvardssm** [reply]
It just became a hype at some point, but ultimately its just another TLD. See https://blog.hubspot.com/website/io-domains

**Reply by RiskySolutions** [reply]
@halvardssm thanks now i understand
